### ISSUE TITLE
Some question about bbox regrssion

### ISSUE THREAD

**Issue by wcy1122** [NVlabs/wetectron#4]
I want to ask some questions about the usage of bbox regression sinc there is no code now
1. during train, did you use the bbox modified by bbox regression for each student stram to the next?
2. how did you use the bbox regress during test time, just average the offset of different stream or some other method?
thanks

**Comment by jason718** [comment]
Thanks for the attention.
1. No, we didn't use the modified bbox to train each student branch during training.
2. yes, we simply average the predictions. We tried different solutions like voting, but avg works the best in our case.

**Comment by jason718** [comment]
close for now. feel free to reopen.

**Comment by wcy1122** [comment]
Thanks for reply at first, and here is some other questions.

1. Is the labels of three bbox regress layers are generated by mil stream, refine_stream 1 and 2 ?
2. You multiply the loss of the first refine layer by 3(see here #3) the same operation at bbox regress layer ?
3. Is the weight of bbox regress loss is 1/N (N is the number of bboxes) for all the trainning time ?

Moreover, if it is possible to release original version of code written on the basis of "Mask R-CNN Benchmark", or at least some important part of it ?

Thanks

**Comment by jason718** [comment]
1. yes
2. yes. because the loss_weights from mil_stream is of different scale (there are two softmax).  
3. yes. It's normalized by the num_rois following supervised work. 
Detectron2 explanation here: https://github.com/facebookresearch/detectron2/blob/ff4aa0d073a6c802e1219722d1378de0701da1a9/detectron2/modeling/roi_heads/fast_rcnn.py#L285

For last question, please email me.
